scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,431 @@ def rvs(self, dim, size=1, random_state=None):
+    broadcast, following the rules above, as if ``m.shape == (2,)``.
+    Thus the result should also be of shape ``(2,)``, but since each output is
+    a :math:`2 \times 2` matrix. The result, in fact, has shape ``(2, 2, 2)``,
+    where ``result[0]`` is equal to
+    ``multivariate_hypergeom.cov(m=[7, 9], n=8)`` and ``result[1]`` is equal
+    to ``multivariate_hypergeom.cov(m=[10, 15], n=12)``.
+
+    See also
+    --------
+    scipy.stats.hypergeom
+    numpy.random.Generator.multivariate_hypergeometric
+
+    References
+    ----------
+    .. [1] The Multivariate Hypergeometric Distribution,
+           http://www.randomservices.org/random/urn/MultiHypergeometric.html
+
+    .. [2] Thomas J. Sargent and John Stachurski, 2020,
+           Multivariate Hypergeometric Distribution
+           https://python.quantecon.org/_downloads/pdf/multi_hyper.pdf
+    """
+    def __init__(self, seed=None):
+        super(multivariate_hypergeom_gen, self).__init__(seed)
+        self.__doc__ = doccer.docformat(self.__doc__, mhg_docdict_params)
+
+    def __call__(self, m, n, seed=None):
+        """
+        Create a frozen multivariate_hypergeom distribution.
+
+        See `multivariate_hypergeom_frozen` for more information.
+        """
+        return multivariate_hypergeom_frozen(m, n, seed=seed)
+
+    def _process_parameters(self, m, n):
+        m = np.asarray(m, dtype=np.int_)
+        n = np.asarray(n, dtype=np.int_)
+        if m.ndim == 0:
+            raise ValueError("'m' must be an array!")
+        mcond = np.any(m < 0, axis=-1)
+
+        M = m.sum(axis=-1)
+
+        ncond = (n <= 0) | (n > M)
+        return M, m, n, mcond | ncond
+
+    def _process_quantiles(self, x, M, m, n):
+        xx = np.asarray(x, dtype=np.int_)
+        if xx.ndim == 0:
+            raise ValueError("x must be an array.")
+        if xx.size != 0 and not xx.shape[-1] == m.shape[-1]:
+            raise ValueError("Size of each quantile should be size of 'm': "
+                             "received %d, but expected %d." %
+                             (xx.shape[-1], m.shape[-1]))
+
+        xcond = np.any(xx != x, axis=-1)
+        xcond = xcond | np.any((xx < 0) | (xx > m), axis=-1)
+        xcond = xcond | (xx.sum(axis=-1) != n)
+        return xx, xcond
+
+    def _checkresult(self, result, cond, bad_value):
+        result = np.asarray(result)
+        if cond.ndim != 0:
+            result[cond] = bad_value
+        elif cond:
+            if result.ndim == 0:
+                return bad_value
+            result[...] = bad_value
+        if result.ndim == 0:
+            return result[()]
+        return result
+
+    def _logpmf(self, x, M, m, n):
+        num = betaln(m+1, 1) - betaln(x+1, m-x+1)
+        den = betaln(M+1, 1) - betaln(n+1, M-n+1)
+        return num.sum(axis=-1) - den
+
+    def logpmf(self, x, m, n):
+        """
+        Log of the Multivariate Hypergeometric probability mass function.
+
+        Parameters
```

Comment:
Is this actually accessible in the rendered documentation? It would sure be nice if it were, but currently this (and a lot of stats methods) look like:
![image](https://user-images.githubusercontent.com/6570539/93002770-14686080-f4ee-11ea-8426-4c4ceb7af9ce.png)
so it's not easy to get to them.

Reply:
It is the same for all the distributions :(

I don't know what should be done with it. Should I go ahead and remove the back ticks?

Reply:
No. I'll file an issue I guess.